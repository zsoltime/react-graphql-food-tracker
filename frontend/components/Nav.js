import Link from 'next/link';
import React from 'react';

import NavStyles from './Nav.styles';

const Nav = () => (
  <NavStyles>
    <li>
      <Link href="/dashboard">
        <a>Dashboard</a>
      </Link>
    </li>
    <li>
      <Link href="/track">
        <a>Diary</a>
      </Link>
    </li>
    <li>
      <Link href="/foods">
        <a>Foods</a>
      </Link>
    </li>
    <li>
      <Link href="/account">
        <a>Account</a>
      </Link>
    </li>
  </NavStyles>
);

export default Nav;
