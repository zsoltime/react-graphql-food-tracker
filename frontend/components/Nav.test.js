import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Nav from './Nav';

describe('<Nav />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Nav />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
