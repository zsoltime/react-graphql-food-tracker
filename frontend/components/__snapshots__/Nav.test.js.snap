// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> matches snapshot 1`] = `
<Navstyles__NavStyles>
  <li>
    <Link
      href="/dashboard"
    >
      <a>
        Dashboard
      </a>
    </Link>
  </li>
  <li>
    <Link
      href="/track"
    >
      <a>
        Diary
      </a>
    </Link>
  </li>
  <li>
    <Link
      href="/foods"
    >
      <a>
        Foods
      </a>
    </Link>
  </li>
  <li>
    <Link
      href="/account"
    >
      <a>
        Account
      </a>
    </Link>
  </li>
</Navstyles__NavStyles>
`;
